cmd/server.go

```diff
@@ -20,54 +20,47 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	"go.infratographer.com/x/crdbx"
-	"go.infratographer.com/x/otelx"
-	"go.infratographer.com/x/versionx"
-
 	"go.infratographer.com/tenant-api/internal/config"
-	"go.infratographer.com/tenant-api/pkg/api"
+	"go.infratographer.com/tenant-api/pkg/api/v1"
 	"go.infratographer.com/tenant-api/pkg/echox"
+	"go.infratographer.com/x/crdbx"
+	"go.infratographer.com/x/otelx"
+	"go.uber.org/zap"
 )
 
 var (
-	APIDefaultListen = "0.0.0.0:7601"
+	APIDefaultListen = ":8080"
```

Comment:
I've seen us use 7601 in a bunch of places now, I'm not sure why but I think it's related to echo?

Reply:
ah ok, i've switched it back to 7601